fe/fe-core/src/main/java/com/starrocks/qe/StmtExecutor.java

```diff
@@ -420,7 +423,38 @@ public void execute() throws Exception {
                 }
             } else if (parsedStmt instanceof DmlStmt) {
                 try {
-                    handleDMLStmt(execPlan, (DmlStmt) parsedStmt);
+                    if (parsedStmt instanceof InsertStmt && ((InsertStmt) parsedStmt).isOverwrite()) {
```

Comment:
Abstract these following  codes to a method.

Reply:
ok